<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>edititemdialog</class>
 <widget class="QDialog" name="edititemdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>617</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>justagain DIN</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>تعديل صنف</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/icons/mainicon.ico</normaloff>:/icons/mainicon.ico</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>381</width>
     <height>391</height>
    </rect>
   </property>
   <property name="title">
    <string>بينات المنتج</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>30</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>justagain DIN</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>رقم البار كود:</string>
       </property>
       <property name="buddy">
        <cstring>barcode</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="barcode">
       <property name="inputMethodHints">
        <set>Qt::ImhNone</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>80</y>
      <width>271</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>إسم المنتج:</string>
       </property>
       <property name="buddy">
        <cstring>title</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="title"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>271</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>وصف المنتج:</string>
       </property>
       <property name="buddy">
        <cstring>desc</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QTextEdit" name="desc"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>299</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_4">
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>السعر:</string>
       </property>
       <property name="buddy">
        <cstring>price</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="price"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>350</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_5">
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>النوع:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="type"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>580</y>
     <width>281</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>تطبيق</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>الغاء</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>182</width>
     <height>92</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <property name="documentMode">
    <bool>false</bool>
   </property>
   <property name="tabsClosable">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="icon">
     <iconset>
      <normalon>:/icons/Search-good-icon.png</normalon>
     </iconset>
    </attribute>
    <attribute name="title">
     <string>الرقم</string>
    </attribute>
    <layout class="QFormLayout" name="formLayout_6">
     <item row="0" column="0">
      <widget class="QLineEdit" name="Numberinput">
       <property name="maxLength">
        <number>999999999</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="icon">
     <iconset>
      <normalon>:/icons/Monitoring-icon.png</normalon>
     </iconset>
    </attribute>
    <attribute name="title">
     <string>بحث</string>
    </attribute>
    <layout class="QFormLayout" name="formLayout_7">
     <item row="0" column="0">
      <widget class="QLineEdit" name="search_item">
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>70</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>تعديل</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>Numberinput</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>title</tabstop>
  <tabstop>desc</tabstop>
  <tabstop>price</tabstop>
  <tabstop>type</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>search_item</tabstop>
  <tabstop>barcode</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>edititemdialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>121</x>
     <y>364</y>
    </hint>
    <hint type="destinationlabel">
     <x>30</x>
     <y>356</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
